code/html/custom.js

```diff
@@ -126,17 +126,18 @@ function loadTimeZones() {
     ];
 
     for (var i in time_zones) {
-        var value = time_zones[i];
-        var offset = value >= 0 ? value : -value;
-        var text = "GMT" + (value >= 0 ? "+" : "-") +
-            zeroPad(parseInt(offset / 60, 10), 2) + ":" +
-            zeroPad(offset % 60, 2);
-        $("select[name='ntpOffset']").append(
-            $("<option></option>").
-                attr("value",value).
-                text(text));
+        if ({}.hasOwnProperty.call(time_zones, i)) {
```

Comment:
`Object.prototype.hasOwnProperty.call` ?
I thinks this creates object each iteration just to call hasownproperty